### ISSUE TITLE
panic: runtime error: slice bounds out of range

### ISSUE THREAD

**Issue by meroujanantonyan** [photoprism/photoprism#116]
can not index originals dir in docker

```yaml
 photoprism:
    image: photoprism/photoprism:latest
    container_name: photoprism
    restart: unless-stopped
    networks:
      br_docker:
        ipv6_address: fd9f::10:0:0:28
        ipv4_address: 10.0.0.28
    expose:
      - "80/tcp"
    environment:
      PHOTOPRISM_READ_ONLY: "true"
      PHOTOPRISM_ORIGINALS_PATH: /srv/photoprism/photos-originals
      PHOTOPRISM_CACHE_PATH: /srv/photoprism/cache
      PHOTOPRISM_SQL_PATH: /srv/photoprism/database
    volumes:
      - /mnt/raid-lv-data/docker-persistent-data/nextcloud/data/vx3r/files/Photos:/srv/photoprism/photos-originals
      - /mnt/raid-lv-data/docker-persistent-data/photoprism/cache:/srv/photoprism/cache
      - /mnt/raid-lv-data/docker-persistent-data/photoprism/database:/srv/photoprism/database
    cap_add:
    - SYS_PTRACE
    security_opt:
    - apparmor:unconfined
```

error

```
docker-compose exec photoprism photoprism index
INFO[2019-06-05T06:48:15Z] indexing photos in /srv/photoprism/photos-originals
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/rwcarlsen/goexif/mknote.(*nikonV3).Parse(0x3a22440, 0xc00053fb00, 0x0, 0x0)
        /go/pkg/mod/github.com/rwcarlsen/goexif@v0.0.0-20180518182100-8d986c03457a/mknote/mknote.go:58 +0x261
github.com/rwcarlsen/goexif/exif.Decode(0x25576c0, 0xc000236158, 0xc000236158, 0x0, 0xc000541500)
        /go/pkg/mod/github.com/rwcarlsen/goexif@v0.0.0-20180518182100-8d986c03457a/exif/exif.go:301 +0x528
github.com/photoprism/photoprism/internal/photoprism.(*MediaFile).ExifData(0xc0000ca6e0, 0x0, 0x0, 0x0)
        /go/src/github.com/photoprism/photoprism/internal/photoprism/exif.go:61 +0x106
github.com/photoprism/photoprism/internal/photoprism.(*Indexer).indexMediaFile(0xc0003b5f20, 0xc0000ca6e0, 0x1, 0x1)
        /go/src/github.com/photoprism/photoprism/internal/photoprism/indexer.go:137 +0x2916
github.com/photoprism/photoprism/internal/photoprism.(*Indexer).IndexRelated(0xc0003b5f20, 0xc0000ca640, 0xc0000ca640)
        /go/src/github.com/photoprism/photoprism/internal/photoprism/indexer.go:304 +0xa1
github.com/photoprism/photoprism/internal/photoprism.(*Indexer).IndexAll.func1(0xc000234140, 0x41, 0x25a5f80, 0xc0004dd040, 0x0, 0x0, 0xce32ea, 0xc0004dd040)
        /go/src/github.com/photoprism/photoprism/internal/photoprism/indexer.go:342 +0xe0
path/filepath.walk(0xc000234140, 0x41, 0x25a5f80, 0xc0004dd040, 0xc0003b5f40, 0x0, 0x0)
        /usr/local/go/src/path/filepath/path.go:358 +0x426
path/filepath.walk(0xc000242ea0, 0x2d, 0x25a5f80, 0xc0004dcf70, 0xc0003b5f40, 0x0, 0x0)
        /usr/local/go/src/path/filepath/path.go:382 +0x300
path/filepath.walk(0xc000242e10, 0x25, 0x25a5f80, 0xc0004dcea0, 0xc0003b5f40, 0x0, 0x0)
        /usr/local/go/src/path/filepath/path.go:382 +0x300
path/filepath.walk(0xc00004401a, 0x20, 0x25a5f80, 0xc0004dcdd0, 0xc0003b5f40, 0x0, 0x20)
        /usr/local/go/src/path/filepath/path.go:382 +0x300
path/filepath.Walk(0xc00004401a, 0x20, 0xc0003b5f40, 0xc937d7, 0xc00046f8e0)
        /usr/local/go/src/path/filepath/path.go:404 +0xff
github.com/photoprism/photoprism/internal/photoprism.(*Indexer).IndexAll(0xc0003b5f20, 0xc0003b5f20)
        /go/src/github.com/photoprism/photoprism/internal/photoprism/indexer.go:327 +0xaf
github.com/photoprism/photoprism/internal/commands.indexAction(0xc000262c60, 0x0, 0xc000262c60)
        /go/src/github.com/photoprism/photoprism/internal/commands/index.go:36 +0x21a
github.com/urfave/cli.HandleAction(0x1fdb1e0, 0x2325a48, 0xc000262c60, 0xc000512600, 0x0)
        /go/pkg/mod/github.com/urfave/cli@v1.20.0/app.go:490 +0xc8
github.com/urfave/cli.Command.Run(0x228453a, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0x22a5261, 0x18, 0x0, ...)
        /go/pkg/mod/github.com/urfave/cli@v1.20.0/command.go:210 +0x996
github.com/urfave/cli.(*App).Run(0xc000524340, 0xc00000e060, 0x2, 0x2, 0x0, 0x0)
        /go/pkg/mod/github.com/urfave/cli@v1.20.0/app.go:255 +0x6af
main.main()
        /go/src/github.com/photoprism/photoprism/cmd/photoprism/photoprism.go:34 +0x2de
```

directory content (not only jpg files)

```
docker-compose exec photoprism ls -ltrha "/srv/photoprism/photos-originals/"
total 72K
drwxr-xr-x  4 www-data www-data 4.0K May 21 02:12 2007
drwxr-xr-x  4 www-data www-data 4.0K May 21 02:12 2008
...
drwxr-xr-x  7 www-data www-data 4.0K May 21 02:30 2019
drwxr-xr-x  2 www-data www-data 4.0K May 21 02:36 unknown
drwxr-xr-x  2 www-data www-data 4.0K Jun  5 01:43 iphone

drwxr-xr-x  1 root     root     4.0K Jun  5 06:33 ..
```

**Comment by lastzero** [comment]
Did you try to index non-image files? We currently check for Exif headers in all files (except XML) and this obviously may fail for random binary / video files.

**Comment by lastzero** [comment]
Are there High Efficiency Image File Format (HEIF) files?

**Comment by lastzero** [comment]
It would really help to have a file that causes this error.

**Comment by lastzero** [comment]
Might be fixed now, see aab504f. Build for master is running... Travis CI automatically pushes new images to Docker Hub.

**Comment by meroujanantonyan** [comment]
Sorry for the late reply. Tryed to reindex, seems to work now. 
Thank you
